hwdb.d/60-keyboard.hwdb

```diff
@@ -637,8 +639,6 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBook*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBookStudioG5*:*
  KEYBOARD_KEY_64=calendar                               # Calendar icon (Fn + F12)
  KEYBOARD_KEY_6d=displaytoggle                          # Display icon
- KEYBOARD_KEY_66=connect                                # Pickup phone button → connect → XF86Go
- KEYBOARD_KEY_65=cancel                                 # Hangup phone button → cancel → Cancel
```

Comment:
these were added explicitly, why are they being removed now? Won't this cause regressions for this G5 device? /cc @hadess

Reply:
> these were added explicitly, why are they being removed now?

[This pull request](https://github.com/systemd/systemd/pull/17525) was submitted before Linux 5.10 when appropriate keycodes wasn't available. 

> Won't this cause regressions for this G5 device?

Fedora 34 and Ubuntu 21.04 with systemd 247 (that include commit 21d8f2533db7c46b80f74bd147869eb4f10195a6) not shipped yet. If this pull request will be accepted, I could submit same path to Ubuntu as backport and @jwrdegoede could backport it to Fedora 34 I guess. 

This will allow to avoid situation we have with Logitech wireless keyboard: https://github.com/systemd/systemd/blob/a3c5d7e95ba3cf6557c7b23e900e37c066fba392/hwdb.d/60-keyboard.hwdb#L942 - since this mapping is present for a long time, I don't think adjusting it now is possible. On other hand with HP ZBook Studio G5 we still have a chance to avoid same outcome and use appropriate mapping almost from the beginning.

Reply:
I see - backporting to https://github.com/systemd/systemd-stable would be safer I guess, Debian will also need it

Reply:
@bluca still not sure why you think that I can help with keyboard or mouse quirks ;)

Reply:
sorry - you seem to know things :-D

Reply:
Thanks for the compliment 😂